src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -167,5 +170,40 @@ public override Task<HttpResponseMessage> ExecuteAsync(HttpControllerContext con
 
             return base.ExecuteAsync(controllerContext, cancellationToken);
         }
+
+        [Route("admin/hook/{name}")]
```

Comment:
This should be admin/hooks/{name} as I had mentioned before (following standard rest naming patterns).